src/wp-includes/class-wp-scripts.php

```diff
@@ -714,6 +781,262 @@ public function in_default_dir( $src ) {
 		return false;
 	}
 
+	/**
+	 * This overrides the add_data method from WP_Dependencies, to support normalizing of $args.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @param string $handle Name of the item. Should be unique.
+	 * @param string $key    The data key.
+	 * @param mixed  $value  The data value.
+	 * @return bool True on success, false on failure.
+	 */
+	public function add_data( $handle, $key, $value ) {
+		if ( 'script_args' === $key ) {
+			$args = $this->get_normalized_script_args( $handle, $value );
+			if ( $args['in_footer'] ) {
+				parent::add_data( $handle, 'group', 1 );
+			}
+			return parent::add_data( $handle, $key, $args );
+		}
+		return parent::add_data( $handle, $key, $value );
+	}
+
+	/**
+	 * Checks all handles for any delayed inline scripts.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @return bool True if the inline script present, otherwise false.
+	 */
+	public function has_delayed_inline_script() {
+		foreach ( $this->registered as $handle => $script ) {
+			// Non standalone scripts in the after position, of type async or defer, are usually delayed.
+			$strategy = $this->get_intended_strategy( $handle );
+			if ( $this->is_non_blocking_strategy( $strategy ) &&
+				$this->has_non_standalone_inline_script( $handle, 'after' )
+			) {
```

Comment:
```suggestion
			if (
				$this->is_non_blocking_strategy( $strategy )
				&& $this->has_non_standalone_inline_script( $handle, 'after' )
			) {
```
For consistency with other multiline conditionals in this PR.

Reply:
Thanks @costdev this has been rectified, resolving this conversation.